lib/elixir/test/elixir/keyword_test.exs

```diff
@@ -42,4 +42,35 @@ defmodule KeywordTest do
       Keyword.get_and_update!([a: 1], :a, fn value -> value end)
     end
   end
+
+  test "merge/2" do
```

Comment:
Everything else here is tested in doctests, it feels a bit weird to only test `merge/2` 😕